pkg/network/netlink/consumer.go

```diff
@@ -337,6 +337,37 @@ func (c *Consumer) dumpTable(family uint8, output chan Event, ns netns.NsHandle)
 	})
 }
 
+func FetchDummyEntry(ns netns.NsHandle) error {
```

Comment:
I think it would be better to name this for what we are using this for, or wrap it another function that actually says that. Otherwise it is a little confusing for someone reading the code